.changes/unreleased/tinymce-TINY-10994-2024-06-17.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: The pattern commans would execute even if command was not supported.
```

Also fix the PR summary
```suggestion
body: The pattern commands would execute even if the command was not enabled.
```